lightcurve_fitting/lightcurve.py

```diff
@@ -534,43 +534,53 @@ def mag2flux(mag, dmag=np.nan, zp=0., nondet=None, nondetSigmas=3.):
         flux[nondet] = 0
     return flux, dflux
 
-
-def binflux(time, flux, dflux, delta=0.2):
+def binflux(time, flux, dflux, delta=0.2, include_zero=True):
     """
     Bin a light curve by averaging points within ``delta`` of each other in time
-
     Parameters
     ----------
     time, flux, dflux : array-like
         Arrays of times, fluxes, and uncertainties comprising the observed light curve
     delta : float, optional
         Bin size, in the same units as ``time``. Default: 0.2
-
+    include_zero : bool, optional
+        Include data points with no error bar
     Returns
     -------
     time, flux, dflux : array-like
         Binned arrays of times, fluxes, and uncertainties
     """
-    bin_time = []
-    bin_flux = []
+    bin_time  = []
+    bin_flux  = []
     bin_dflux = []
     while len(flux) > 0:
         grp = np.array(abs(time - time[0]) <= delta)
-        time_grp = time[grp]
-        flux_grp = flux[grp]
+        time_grp  = time[grp]
+        flux_grp  = flux[grp]
```

Comment:
Can you undo all these whitespace changes? PEP-8 asks for two blank lines between functions and only one space before and after `=`. I think the blank lines in the docstring might be required to build the online documentation with Sphinx (although I haven't checked what happens if they're not there).

Reply:
done